<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayRecordFilewidget</class>
 <widget class="QWidget" name="PlayRecordFilewidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>690</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Yahei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>playForm</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/res/tz.ico</normaloff>:/icon/res/tz.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widgetPlayRecord" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>690</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widgetPlayRecord
{
	background-color: rgba(160, 160, 160, 210);
	font-size:18px;
}</string>
   </property>
   <widget class="QPushButton" name="btnPlay">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>633</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnPlay
{
	
	border-image: url(:/png/video_play.png);
}
QPushButton#btnPlay:hover
{
	
	border-image: url(:/png/video_play_focus.png);
}
QPushButton#btnPlay:pressed
{
	border-image: url(:/png/video_play_focus.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>4</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;关闭窗口&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: rgb(114, 187, 255);
	background-color: rgba(255, 255, 255, 0);
	border-color: rgba(255, 255, 255, 0);
	font-size: 30px;
}

QPushButton:hover
{
   color: rgb(255, 255, 255);
}
QPushButton:pressed
{
   color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>4</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#labelName
{
	font-size:20px;
	font-family:Microsoft Yahei;	
}</string>
    </property>
    <property name="text">
     <string>视频文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>633</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnStop
{
	
	border-image: url(:/png/video_stop.png);

}
QPushButton#btnStop:hover{	border-image: url(:/png/video_stop_focus.png);}
QPushButton#btnStop:pressed{	
	border-image: url(:/png/video_stop_focus.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="Playtime">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>609</y>
      <width>800</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A1A1A1, stop:1 #B4B4B4);
    margin: 2px 0 2px;
}

QSlider::handle:horizontal {
	border-image: url(:/png/video_progress_button.png);

    width: 16px;
 	height: 12px;
    margin: -3px 0px -7px 0px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */

}

QSlider::sub-page:horizontal{ 	
	height: 8px;
	background-color: rgb(255, 255, 255);
	margin: 2px 0px 2px 0px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>800</width>
      <height>558</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QMdiArea
{
	background-color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
